skimage/viewer/tests/test_plugins.py

```diff
@@ -22,7 +24,7 @@ def setup_line_profile(image, limits='image'):
     return plugin
 
 
-@testing.skipif(not has_qt, reason="Qt not installed")
+@pytest.mark.skipif(not has_qt, reason="Qt not installed")

```

Comment:
```suggestion
@pytest.mark.skipif(not has_qt, reason="Needs Qt")
```
out of consistency? (in the following as well...)